frontend/src/services/exchangeRateService.ts

```
@@ -51,29 +51,72 @@ interface HistoricalExchangeRateResponse extends CurrencyPair {
+  return {
+    baseCurrencyCode,
+    quoteCurrencyCode,
+    data: allData,
+  };
+};
+
+const _getHistoricalExchangeRates = async (
+  baseCurrencyCode: string,
+  quoteCurrencyCode: string,
+  startDate?: ProperDate,
+  page: number = 1,
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)